cmd/generate.go

```diff
@@ -178,9 +178,12 @@ func genInit() {
 func genSDKInit() {
 	genSDKCmd.Flags().StringP("lang", "l", "go", fmt.Sprintf("language to generate sdk for (available options: [%s])", strings.Join(SDKSupportedLanguageTargets(), ", ")))
 
-	genSDKCmd.Flags().StringP("schema", "s", "./openapi.yaml", "path to the openapi schema")
+	genSDKCmd.Flags().StringP("schema", "s", "./openapi.yaml", "local filepath or URL for the OpenAPI schema")
```

Comment:
I'll leave it up to you but wondering it might make the code cleaner if there were separate `schema` and `schema_url` flags?

Reply:
Think that burdens the user a bit since they'll have to choose the right flag. Also if we want to refactor systems that use this (ie. action), we can update to run CLI without having to do prerequisite checks. If we have different flags, then code would still have to be in the action to check if it's a file path vs. URL. I like the abstraction of having that checking be in the CLI so we can just call it.